Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -0,0 +1,1203 @@
+            {
+                lock (_intLock)
+                {
+                    return _lanSubnets.Contains(ep) && !_excludedSubnets.Contains(ep);
+                }
+            }
+
+            return false;
+        }
+
+        /// <inheritdoc/>
+        public bool IsInLocalNetwork(IPAddress address)
+        {
+            if (address == null)
+            {
+                throw new ArgumentNullException(nameof(address));
+            }
+
+            // See conversation at https://github.com/jellyfin/jellyfin/pull/3515.
+            if (TrustAllIP6Interfaces && address.AddressFamily == AddressFamily.InterNetworkV6)
+            {
+                return true;
+            }
+
+            lock (_intLock)
+            {
+                // As private addresses can be redefined by Configuration.LocalNetworkAddresses
+                return _lanSubnets.Contains(address) && !_excludedSubnets.Contains(address);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsPrivateAddressRange(IPObject address)
+        {
+            if (address == null)
+            {
+                throw new ArgumentNullException(nameof(address));
+            }
+
+            // See conversation at https://github.com/jellyfin/jellyfin/pull/3515.
+            if (TrustAllIP6Interfaces && address.AddressFamily == AddressFamily.InterNetworkV6)
+            {
+                return true;
+            }
+            else
+            {
+                return address.IsPrivateAddressRange();
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsExcludedInterface(IPAddress address)
+        {
+            lock (_intLock)
+            {
+                if (_bindExclusions.Count > 0)
+                {
+                    return _bindExclusions.Contains(address);
+                }
+
+                return false;
+            }
+        }
+
+        /// <inheritdoc/>
+        public NetCollection GetFilteredLANSubnets(NetCollection? filter = null)
+        {
+            lock (_intLock)
+            {
+                if (filter == null)
+                {
+                    return NetCollection.AsNetworks(_lanSubnets.Exclude(_excludedSubnets));
+                }
+
+                return _lanSubnets.Exclude(filter);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsValidInterfaceAddress(IPAddress address)
+        {
+            lock (_intLock)
+            {
+                return _interfaceAddresses.Contains(address);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool TryParseInterface(string token, out IPNetAddress result)
+        {
+            if (string.IsNullOrEmpty(token))
+            {
+                result = IPNetAddress.None;
+                return false;
+            }
+
+            if (_interfaceNames != null && _interfaceNames.TryGetValue(token.ToLower(CultureInfo.InvariantCulture), out int index))
```

Comment:
`_interfaceNames` is not nullable

Reply:
Used to get rid of intellisense message.